### ISSUE TITLE
conda skeleton pypi throws xmlrpc.client.Fault: <Fault -32700: 'parse error; not well formed'>

### ISSUE THREAD

**Issue by ssbarnea** [conda/conda-build#1207]
I tried to do `conda skeleton pypi jira` and I got:
```
Traceback (most recent call last):
  File "/Users/sorins/anaconda3/bin/conda-skeleton", line 5, in <module>
    sys.exit(main())
  File "/Users/sorins/anaconda3/lib/python3.5/site-packages/conda_build/main_skeleton.py", line 300, in main
    args_func(args, p)
  File "/Users/sorins/anaconda3/lib/python3.5/site-packages/conda_build/main_build.py", line 389, in args_func
    args.func(args, p)
  File "/Users/sorins/anaconda3/lib/python3.5/site-packages/conda_build/main_skeleton.py", line 315, in execute
    pypi.main(args, parser)
  File "/Users/sorins/anaconda3/lib/python3.5/site-packages/conda_build/pypi.py", line 263, in main
    all_packages = [match["name"] for match in client.search({"name": args.packages}, "or")]
  File "/Users/sorins/anaconda3/lib/python3.5/xmlrpc/client.py", line 1092, in __call__
    return self.__send(self.__name, args)
  File "/Users/sorins/anaconda3/lib/python3.5/xmlrpc/client.py", line 1432, in __request
    verbose=self.__verbose
  File "/Users/sorins/anaconda3/lib/python3.5/site-packages/conda_build/pypi.py", line 233, in request
    return self.parse_response(resp)
  File "/Users/sorins/anaconda3/lib/python3.5/site-packages/conda_build/pypi.py", line 242, in parse_response
    return u.close()
  File "/Users/sorins/anaconda3/lib/python3.5/xmlrpc/client.py", line 655, in close
    raise Fault(**self._stack[0])
xmlrpc.client.Fault: <Fault -32700: 'parse error; not well formed'>
```
`conda info` returns:
```
Current conda install:

             platform : osx-64
        conda version : 4.1.11
    conda-env version : 2.5.2
  conda-build version : 1.21.3
       python version : 3.5.2.final.0
     requests version : 2.10.0
     root environment : /Users/sorins/anaconda3  (writable)
  default environment : /Users/sorins/anaconda3
     envs directories : /Users/sorins/anaconda3/envs
        package cache : /Users/sorins/anaconda3/pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/osx-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/osx-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : /Users/sorins/.condarc
         offline mode : False
    is foreign system : False
```

**Comment by ssbarnea** [comment]
This seems to be related to https://github.com/pypa/warehouse/issues/1417

**Comment by msarahan** [comment]
Indeed, you found the issue, but please see https://github.com/conda/conda-build/issues/1177#issuecomment-238033029 for fix (update conda-build to >=1.21.11)
